src/Accordion/Accordion.tsx

```diff
@@ -27,35 +27,42 @@ export interface AccordionProps {
     iconRotation?: AccordionIconRotation;
     id?: string;
     list: Array<AccrodionListItem>;
+    alternative?: boolean;
 }
 
 const Accordion: React.FunctionComponent<AccordionProps> = (props: AccordionProps) => {
     const [active, setActive] = React.useState<number>(null);
-    const [className, setClassName] = React.useState<string>("custom-accordion");
+    const [itemClassName, setItemClassName] = React.useState<string>("custom-accordion");
     const [idList, setIdList] = React.useState<Array<string>>([]);
 
     React.useEffect(() => {
         constructIds();
-        constructClassName();
+        constructItemClassName();
     }, []);
```

Comment:
@yousifalraheem why doesn't this have a dept too? Also how is it different from the useEffect below ?

Reply:
An empty array means it will be executed when component mounts, I guess we don't need this since the next `useEffect` also gets called when component mounts. But still, this is @kherP's pull-request and I can solve it on my branch not this one. Thanks for noticing.

Reply:
There is a big bug in here caused by not passing the dependencies to the dependency array of the React.useEffect. Since constructClassName() is being used inside the Effect and is reading from props like `props.iconPosition` and `props.iconLocation` they should be added to the dependencies. So basically it should look like this: 

```
    React.useEffect(() => {
        constructIds();
        constructClassName();
        constructItemClassName();
    }, [props.iconPosition, props.iconLocation]);
```

otherwise the effect will not be run again if iconPosition or location props change

You can read more in depth about it in this wonderful article by Dan Abramov:
https://overreacted.io/a-complete-guide-to-useeffect/#what-happens-when-dependencies-lie

Reply:
Oh I see you added after I commented lol

Reply:
removed first useEffect as 2nd useEffect will call the same functions